e-commerce/client/src/App.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { useState } from 'react';
+
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+import Navbar from './components/Navbar';
+import ProductCard from './components/ProductCard';
+import Cart from './components/Cart';
+
+import { useQuery } from 'react-query';
```

Comment:
So here's what we're gonna do. We're going to use `graphql-codegen` to generate types for the queries. You'll need to learn how to create query files, and configure graphql codegen for react-query.

Reply:
Done